hmpps-prisoner-search-indexer/helm_deploy/hmpps-prisoner-search-indexer/values.yaml

```diff
@@ -97,8 +97,13 @@ cron:
   retry_dlqs_schedule: "*/10 * * * *"
   index_size_schedule: "*/10 * * * *"
   check_index_schedule: "*/15 * * * *"
-  index_refresh_schedule: "05 20 * * 1,3,5"
+  index_refresh_schedule: "05 19 * * 1,3,5"
```

Comment:
need to make this a bit earlier so doesn't conflict with prisoner offender search